packages/infra/lib/lambdas-nested-stack.ts

```diff
@@ -1030,6 +1053,74 @@ export class LambdasNestedStack extends NestedStack {
     return acmCertificateMonitorLambda;
   }
 
+  private setupAlohrSftpIngestionLambda(ownProps: {
+    lambdaLayers: LambdaLayers;
+    vpc: ec2.IVpc;
+    secrets: Secrets;
+    config: EnvConfig;
+    alohrSftpIngestionBucket: s3.IBucket;
+    alarmAction: SnsAction | undefined;
+  }): Lambda {
+    const envType = ownProps.config.environmentType;
+    const alohrProps = ownProps.config.hl7Notification?.AlohrSftpIngestionLambda;
+    if (!alohrProps) {
+      throw new Error("AlohrSftpIngestionLambda is undefined in config.");
+    }
+
+    const sftpPasswordSecret = ownProps.secrets["ALOHR_INGESTION_PASSWORD"];
+
+    if (!ownProps.config.hl7Notification) {
+      throw new Error("HL7Notification is undefined in config.");
+    }
+    const queue = ownProps.config.hl7Notification.notificationWebhookSenderQueue;
+
+    if (!queue) {
+      throw new Error("HL7NotificationWebhookSenderQueue is undefined in config.");
+    }
+
+    if (!sftpPasswordSecret) {
+      throw new Error("ALOHR_INGESTION_PASSWORD is not defined in config.");
+    }
+
+    const sftpConfig = alohrProps.sftpConfig;
+    const lambdaTimeout = Duration.minutes(5);
```

Comment:
nit; id default to inlining values if its obvious what they are by looking at them.